deepmd/README.md

```diff
@@ -0,0 +1,23 @@
+# DEEPMD
+
+DeePMD (Deep Potential Molecular Dynamics) [1] is a package for easily building performant neural network potentials for carrying out molecular dynamics. The DeePMD code base has been developed such that trained models can be interfaced directly to LAMMPS.
+
+## Notes
+
+Here we provide two sets of examples, one for training a neural network potential using DeePMD, and a second for running a LAMMPS simulation using that trained model.
+We note that in order to have simple examples the model in question is trained with a small amount of data for a short period of time. Consequently, the accuracy of the resulting model is lower than we should expect for a model trained for longer on a larger amount of data.
+
+### Training
+
+The training script trains a model on the water data provided in the package examples [2] and then freezes that model for production. The job must be submitted from the same directory as the job.pbs file.
```

Comment:
How do we generate the water data? Should we add the ab-initio step?

Reply:
Currently we're just taking their water data, I can look to see if they document how they produced in and if input files are available I can make another cli example to generate an equivalent set of data. However I think that it's important to bundle the data here for this example.